lizard_auth_client/views.py

```diff
@@ -127,6 +127,18 @@ def get(self, request, *args, **kwargs):
         return HttpResponseRedirect(sso_after_login_next)
 
 
+class LocalNotLoggedInView(View):
+    """
+    The user has returned from the SSO server without logging in.
+    Return him to his original page, unauthenticated.
+    """
+    def get(self, request, *args, **kwargs):
+        """Redirect the user to the stored "next" url."""
+        sso_after_login_next = request.session.pop(
+            'sso_after_login_next', '/')
```

Comment:
Er is een setting voor waar je naartoe gaat als je wel ingelogd bent (LOGIN_REDIRECT_URL), maar deze is voor als je niet ingelogd bent, daarom vond ik het wat krom die te gebruiken. Maar ik kan geen scenario verzinnen waarbij next hier niet geset is.